### ISSUE TITLE
autoshare new subalbums.

### ISSUE THREAD

**Issue by davidleeowens2nd** [petitssuisses/piwigo-ShareAlbum#97]
We create new sub-albums within shared albums. It would be an excellent addition if somehow new folders/albums of photos that are added as subalbums of a shared album could have their permissions set automatically. Currently, we have to update permissions each time we add a subalbum and it is leading to clients not seeing most recent albums shared with them if we forget to update permissions.

**Comment by petitssuisses** [comment]
You're right, I shall implement a trigger for new albums which are already shared. I'll mark this one as an enhancement request, to be coded soon.
